src/server/scripts/Commands/cs_tele.cpp

```diff
@@ -35,6 +35,9 @@ EndScriptData */
 #include "Player.h"
 #include "RBAC.h"
 #include "WorldSession.h"
+#include <boost/tokenizer.hpp>
```

Comment:
We have a Tokenizer class in Util.h

Reply:
Shauren why do we have it / what are the advantages of it?
from quickly I looking I can see one advantage for boost, which is it parses the data on demand as the tokens are accessed rather than the constructor, and it only support multiple characters.

Reply:
Not using boost is the advantage - compile times don't suffer

Generally if we have code for it, it should be preferred over directly using 3rd party libraries

Reply:
I was talking about parsing that's done in runtime, not compile time.

And I disagree with that; boost isn't just any 3rd party library, it's widely known and used. I'd even argue that if there are any bugs in the code, it's more likely in ours than theirs, since their product is more widely used and tested.
We're just reinventing the wheel here.

But either way I removed it.